python/example_code/healthlake/health_lake_wrapper.py

```diff
@@ -549,9 +549,22 @@ def wait_export_job_complete(self, datastore_id: str, job_id: str) -> None:
             )
 
     def health_lake_demo(self) -> None:
-        use_smart_on_fhir_data_store = True
+        use_smart_on_fhir_data_store = False
+
+        # Change the following variables to match your environment.
+        datastore_name = "health_imaging_datastore"
+        import_job_name = "my_import_job"
+        input_s3_uri = (
+            "s3://amzn-s3-demo-bucket/import/examples/patient_example.json"
+        )
+        kms_key_id = "arn:aws:kms:us-east-1:123456789012:key/b7f645cb-e564-4981-8672-9e012d1ff1a0"
+        data_access_role_arn = (
+            "arn:aws:iam::123456789012:role/healthlake_access"
+        )
+        export_job_name = "my_export_job"
+        output_s3_uri = "s3://amzn-s3-demo-bucket/export/output/"
```

Comment:
Isn't the standard to let the demo run without making the user update the code? Something like env vars or command line args or a config file?

Reply:
This is not a basics. If the parameters were fewer and simpler, I would pass args. I think this is much more user friendly.